tools/pyi/gen_pyi.py

```diff
@@ -151,6 +151,7 @@ def type_to_python(typename, size=None):
         'Dimname': 'Union[str, ellipsis, None]',
         'DimnameList': 'Sequence[Union[str, ellipsis, None]]',
         'QScheme': '_qscheme',
+        'ArrayRef<double>' : 'ArrayRef<double>'
```

Comment:
Is this correct?

Reply:
No. The output type must be a valid mypy type. You probably want something like `Sequence[float]`

Reply:
Updated